Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11.6-alpine3.18
+LABEL maintainer="savenya2013@gmail.com"
+
+ENV PYTHONDONTWRITECODE=1
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
The `COPY requirements.txt requirements.txt` command assumes that `requirements.txt` is in the same directory as the Dockerfile. Ensure that this file exists in the correct location.